### DISCUSSION TITLE
Pollutant Transport in Shallow-water Flows: Application to the Niger River in Bamako

### DISCUSSION THREAD

**Discussion by asamake** [feelpp/feelpp#2224] [Portfolio]
# **Introduction**

We propose a unified framework for the mathematical modeling and numerical simulation of the pollution of the Niger River in Bamako in order to contribute to a better understanding of the dispersion pattern of water pollutants. In terms of specific contribution to the current literature, this paper provides the scientific community interested in water pollution issues with a non-trivial application based on new robust algorithms and advanced numerical tools. The model we are interested in consists of a coupling of the shallow-water equations, which govern unsteady water flows in rivers, with an advection-diffusion equation for pollutant transport in water flows.

# **Numerical Results**

[![Flow discharge](https://i3.ytimg.com/vi/7yM0qPqfwUE/maxresdefault.jpg)](https://youtu.be/7yM0qPqfwUE)

[![Flow discharge](https://i3.ytimg.com/vi/g8JnWS-5Du0/maxresdefault.jpg)](https://youtu.be/g8JnWS-5Du0)

[![Flow discharge](https://i3.ytimg.com/vi/55tizwwfVZI/maxresdefault.jpg)]( https://youtu.be/55tizwwfVZI )

**Comment by nayeem1704042** [comment]
Hi, 
Have you written the code for the model to find out the Pollutant Transport in Shallow-water Flow?
 I was also trying the same, but for my case, the pollutant is assumed as microplastics particles. And I was also doing it for 1D. 
 
 Can you help me out with the hydro dynamic part of the model?
